idaes/__init__.py

```diff
@@ -6,6 +6,12 @@
 import os
 
 from . import config as cfg
```

Comment:
I don't know why I wrote it like this, but I probably had a reason at some point then changed this around.  As it is, it doesn't make much sense.

We can probably just just change it to ```from . import config``` then change the few ```cfg```'s to config.

Reply:
Sounds good, I've made this change and things seem to work just fine.